Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'jquery-rails'
 gem 'jquery-ui-sass-rails'
 gem 'jquery-datetimepicker-rails'
 gem 'json', '~> 1.8', '>= 1.8.3'
+gem 'lazy_high_charts'
```

Comment:
- Actually, you do not need to add this gem but it is not your fault since we did not mention clearly in requirement.
- The `highchart` javascript library has already been added in Expertiza. See [link].(https://github.com/expertiza/expertiza/blob/master/bower.json#L31)